package.json

```diff
@@ -10,11 +10,11 @@
 		"url": "sindresorhus.com"
 	},
 	"engines": {
-		"node": ">=6"
+		"node": ">=10"
```

Comment:
I did this as a separate commit to make it clear to us that the next version will require a major bump.

https://github.com/sindresorhus/on-change/commit/b5e2678c48f82eb3ba863e3964d25d261c7fe27b